docs/articles/documentation/using-testcafe/programming-interface/Runner.md

```diff
@@ -0,0 +1,228 @@
+
+Parameter     | Type   | Description
+------------- | ------ | ----------------------------------
+`testName`    | String | The name of the test.
+`fixtureName` | String | The name of the test fixture.
+`fixturePath` | String | The path to the test fixture file.
+
+#### Example
+
+```js
+runner.filter((testName, fixtureName, fixturePath) => {
+    return fixturePath.startsWith('D') &&
+        testName.match(someRe) &&
+        fixtureName.match(anotherRe);
+});
+```
+
+### <a class="anchor" name="browsers"></a>browsers(browser) → this
+
+### browsers([browser, ...]) → this
+
+Configures the test runner to run tests in the specified browsers.
+
+Each `browser` parameter can be one of the following.
+
+Type                                                                                                | Description                                                                                    | Local/Remote
+--------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | --------------
+String                                                                                              | The browser alias: `"ie"`, `"ff"`, `"chrome"`, `"chromium"`, `"opera"`, `"safari"` or `"edge"` | Local browser
+String                                                                                              | The path to the browser executable.                                                            | Local browser
+`{path: String, cmd: String}`                                                                       | The path to the browser executable (`path`) with command line parameters (`cmd`).              | Local browser
+[BrowserConnection](/testcafe/documentation/using-testcafe/programming-interface/BrowserConnection/)| The remote browser connection.                                                                 | Remote browser
+
+You are free to mix different types of objects in one function call. The `browsers` function concatenates the settings when called several times.
+
+#### Examples
+
+##### Using browser aliases
+
+```js
+runner.browsers(["firefox", "chrome"]);
+```
+
+##### Using the path to the browser executable
+
+```js
+runner.browsers("C:\\Program Files\\Internet Explorer\\iexplore.exe");
+```
+
+##### The path to the browser executable with command line parameters
+
+```js
+runner.browsers({
+    path: "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe",
+    cmd: "--new-window"
+});
+```
+
+##### The remote browser connection
+
+```js
+let createTestCafe   = require('testcafe');
+let testCafe         = await createTestCafe('localhost', 1337, 1338);
+
+let remoteConnection = testcafe.createBrowserConnection();
+
+// Outputs remoteConnection.url to visit it from the remote browser.
+console.log(remoteConnection.url);
+
+remoteConnection.once('ready', () => {
```

Comment:
You write [above](https://github.com/DevExpress/testcafe/pull/298/files?short_path=679a3d4#diff-8726dc00e9fc3a3a0affaafefa409693R47) there is no need to wait for this event usually. Maybe we should not use it in our examples then? Just
```js
await testCafe
    .createRunner()
    .browsers(remoteConnection)
    .run()
```
?